src/components/ContactInfo/index.jsx

```diff
@@ -8,26 +8,17 @@ export const ContactInfo = () => {
   const [socialData, setSocialData] = useState(null);
 
   useEffect(() => {
-    charityAPI('/main-contacts')
-      .then(({ data }) => {
-        setContactData(data);
-      })
-      .catch(error => {
-        console.log(error);
-      });
+    Promise.all([
+      charityAPI('/main-contacts'),
+      charityAPI('/socialmedias')
+    ]).then(values => {
```

Comment:
1. you may take array and object destructuring to the next level 😉
```js
).then(([{data: contact}, {data: socialmedia}]) => {
    setContactData(contacts);
    setSocialData(socialmedia);
})
```

2. please handle error cases (catch)